opensearch-project/OpenSearch-Dashboards
# [Multiple DataSource] Codebase maintenance involves updating typos and removing unused imported packages

### Description


Codebase maintenance involves updating typos and removing unused imported packages.

### Issues Resolved



#6142

## Screenshot



https://github.com/opensearch-project/OpenSearch-Dashboards/assets/139305463/e51240e5-cc54-43b1-8102-393ca5f5f1f4


## Testing the changes


In this video, we test Username and Password auth type by executing following steps:
1. Fill in title, url, auth info into dataSource creation form 
2. Click on "Test connection" and test connection success
3. Click on "Create DataSource" and dataSource creation success
4. Navigate to home page and successfully add sample data into newly created dataSource 
5. Switch to dashboard tab and able to see dashboards successfully

Also test registered auth type locally and works fine. Verified no regressions.

Thanks

### Check List
